packages/firebase_messaging/android/src/main/java/io/flutter/plugins/firebasemessaging/FlutterFirebaseMessagingService.java

```diff
@@ -42,4 +128,148 @@ public void onNewToken(String token) {
     intent.putExtra(EXTRA_TOKEN, token);
     LocalBroadcastManager.getInstance(this).sendBroadcast(intent);
   }
+
+  public static void startBackgroundIsolate(Context context, long callbackHandle) {
+    FlutterMain.ensureInitializationComplete(context, null);
+    String mAppBundlePath = FlutterMain.findAppBundlePath(context);
+    FlutterCallbackInformation flutterCallback =
+        FlutterCallbackInformation.lookupCallbackInformation(callbackHandle);
+    if (flutterCallback == null) {
+      Log.e(TAG, "Fatal: failed to find callback");
```

Comment:
`E/FlutterFcmService: Fatal: failed to find callback`

Logcat logs this both on App in Background and App Terminated.

It could be that the shared preference of the handle call back doesn't exist. Don't know why I'm getting null on the callback.

Reply:
That wold indicate that the Dart side did not set the handle, so the lookup is failing. That is my first feeling, I'll dig deeper.

Reply:
@XSiyabonga I will add the package information about the Application class to the README, thanks.

Reply:
Tested commit #52a19bacbeadac4d0e989091d3ad58e4da3368b1
- run example :
got E/FlutterFcmService(10271): Fatal: failed to find callback
- press "home" android button to hide
- send with firebase console message 
- notification appears but no callback with myBackgroundMessageHandler...
seems there is a bug still.. how to fix?

Reply:
> That wold indicate that the Dart side did not set the handle, so the lookup is failing. That is my first feeling, I'll dig deeper.

how to fix? added below my comment how to reproduce